src/workbench-jupyter-parabricks/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Use the official NVIDIA Clara Parabricks container as the base image
+FROM nvcr.io/nvidia/clara/clara-parabricks:4.5.1-1
+
+ARG NB_USER=jupyter
+ARG NB_UID=1010
+ARG NB_GID=1000
+ARG WORKDIR=/home/${NB_USER}
+
+# Switch to root user to install packages and create new user
+USER root
+
+RUN rm -rf /var/lib/apt/lists/*
+
+# Only create the user, not the group—GID 1000 already exists as 'ubuntu'
+RUN groupadd --gid ${NB_GID} ${NB_USER} && useradd --uid ${NB_UID} --gid ${NB_GID} --create-home --home-dir ${WORKDIR} --shell /bin/bash ${NB_USER}
+
+# Fix ownership for common dirs
+RUN mkdir -p /workspace \
+ && chown -R ${NB_UID}:${NB_GID} ${WORKDIR} /workspace /tmp \
+ && chown -R ${NB_UID}:${NB_GID} /opt/conda || true
```

Comment:
instead of the sudo-passwordless.sh, you can add it in here

RUN echo "${NB_USER} ALL=(ALL) NOPASSWD:ALL" > /etc/sudoers.d/${NB_USER} \
 && chmod 0440 /etc/sudoers.d/${NB_USER}